tests/test_backend.py

```diff
@@ -539,15 +541,21 @@ class TestBackendCommandArgumentParser(unittest.TestCase):
     def test_argument_parser(self):
         """Test if an argument parser object is created on initialization"""
 
-        parser = BackendCommandArgumentParser()
+        parser = BackendCommandArgumentParser(categories=MockedBackendCommand.BACKEND.CATEGORIES)
         self.assertIsInstance(parser.parser, argparse.ArgumentParser)
 
+    def test_categories(self):
+        """Test whether _categories is initialised"""
```

Comment:
Please, add an empty line between the comment and the code. In addition, I guess initialised should be initialized (a quick search on the perceval repo shows that `initialized` is the preferred option, https://github.com/chaoss/grimoirelab-perceval/search?q=initialized&unscoped_q=initialized)

Reply:
Done, thanks :)